static/src/js/components/CollectionDetails/RelatedUrlsModal.js

```diff
@@ -8,29 +8,17 @@ import { pluralize } from '../../util/pluralize'
 
 import './RelatedUrlsModal.scss'
 
-export class RelatedUrlsModal extends Component {
-  constructor(props) {
-    super(props)
-    this.onModalClose = this.onModalClose.bind(this)
-  }
+export const RelatedUrlsModal = ({
+  collectionMetadata,
+  isOpen,
+  onToggleRelatedUrlsModal
+}) => {
+  const { relatedUrls = [] } = collectionMetadata
 
-  onModalClose() {
-    const { onToggleRelatedUrlsModal } = this.props
-    onToggleRelatedUrlsModal(false)
-  }
-
-  render() {
-    const {
-      collectionMetadata,
-      isOpen
-    } = this.props
-
-    const { relatedUrls = [] } = collectionMetadata
-
-    const body = (
-      // eslint-disable-next-line react/jsx-no-useless-fragment
-      <>
-        {
+  const body = (
+    // eslint-disable-next-line react/jsx-no-useless-fragment
+    <>
```

Comment:
We dont need a fragment here

Reply:
Because of how its getting passed down to `EDSCModal` it was getting messier and making unrelated changes we opted to not do this now. Looks like there are other places where we have this as well.